# Generated with epl_ros_generator.py

# openPOWERLINK variables:
string shm_name
uint16 pid

# xap.h variables:
# uint8 DigitalOutput
int64 mm_x_pos
int64 mm_y_pos
int64 mm_z_pos
int64 mm_x_orient
int64 mm_y_orient
int64 mm_z_orient
int64 mm_w_orient
int64 odom_lwheel
int64 odom_rwheel
